macros/hardware_functions/status_leds.cfg

```
@@ -96,6 +96,9 @@ gcode:
 [gcode_macro STATUS_LEDS]
 gcode:
     {% set color = params.COLOR|default('off')|lower %}
+    {% set logo_leds_name = printer["gcode_macro _USER_VARIABLES"]["status_leds_logo_led_name"] %}
+    {% set nozzle_leds_name = printer["gcode_macro _USER_VARIABLES"]["status_leds_nozzle_led_name"] %}
+    {% set logo_transmit = 0 if (logs_leds_name == nozzle_leds_name) else 1 %}
```

I think there is a mistake in the variable name. Can you fix it? I'll merge it when it's done :) 
```suggestion
    {% set logo_transmit = 0 if (logo_leds_name == nozzle_leds_name) else 1 %}
```

---

Ah, nice catch. Fixed!